test/python_output9.py

```
@@ -0,0 +1,2 @@
+data = open('new_file')
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)